sources/org.osbuild.dnf

```diff
@@ -1,9 +1,18 @@
 #!/usr/bin/python3
```

Comment:
Could this be dropped completely?

Reply:
The whole stage you mean? That would be great!

Reply:
Yes